src/ape_test/provider.py

```diff
@@ -117,13 +117,17 @@ def priority_fee(self) -> int:
 
     def send_call(self, txn: TransactionAPI, **kwargs) -> bytes:
         data = txn.dict(exclude_none=True)
-        if "gas" not in data or data["gas"] == 0:
-            data["gas"] = int(1e12)
-
         block_id = kwargs.pop("block_identifier", None)
         state = kwargs.pop("state_override", None)
         call_kwargs = {"block_identifier": block_id, "state_override": state}
-        tx_params = cast(TxParams, txn.dict())
+
+        # Remove unneeded properties
+        data.pop("gas", None)
```

Comment:
I noticed this while testing... Is this ok? @fubuloubu 
I don't see a point in having `gas` values in calls and it sometimes causes issues when it is